pkg/bootstrapper/bootstrapper.go

```diff
@@ -124,6 +124,9 @@ type winNodeBootstrapper struct {
 	svcMgr *mgr.Mgr
 	// installDir is the directory the the kubelet service will be installed
 	installDir string
+	// logDir is the directory that captures log outputs that are to be made available in Artifacts
```

Comment:
Sorry for not catching this earlier, logDir should be limited to kubelet, technically it is the directory that captures logs of kubelet.